src/index.js

```diff
@@ -34,6 +34,40 @@ export function toggleCreator(context) {
   }
 }
 
+export function handleSumbit(func){
+  return (e) => {
+    e.preventDefault();
+
+    const form = e.target;
+    let values = {};
+
+    let children = [];
+
+    children.push(form.elements);
+
+    for(let element of children[0]){
```

Comment:
why bother with a children array if just accessing the first element of it? 

you could do `form.elements.forEach(...)`?